scripts/utilities/get_curl_url.py

```
@@ -0,0 +1,9 @@
+import requests
+
+file_path = r"C:\Users\jacob\OneDrive - UBC\Desktop\Personal Projects\D&D Battlemaps\dnd_battlemaps_lora_dataset_20250812_233504.zip"
+with open(file_path, "rb") as f:
+    response = requests.put(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)